src/overlay/widgets/Console.cpp

```diff
@@ -149,9 +149,20 @@ void Console::Log(const std::string& acpText)
     std::lock_guard<std::recursive_mutex> _{ m_outputLock };
 
     size_t first = 0;
-    while (first < acpText.size())
+    size_t size = acpText.size();
+    while (first < size)
     {
-        const auto second = acpText.find_first_of('\n', first);
+        // find_first_of \r or \n
+        size_t second = std::string::npos;
```

Comment:
should have probably be std::string_view::npos
not like it matters much in the end, but for consistency sake